### PR TITLE
OSSM-7054 OSSM 2.6.1 (At Stage): [DOC] Release Notes, Known Issues and Bug Fixes

### PR DESCRIPTION
**Pull request by gwynnemonahan**
[OSSM-7054](https://issues.redhat.com//browse/OSSM-7054) OSSM 2.6.1 (At Stage): [DOC] Release Notes, Known Issues and Bug Fixes

Version(s):

OCP 4.14+

Issue:

https://issues.redhat.com/browse/OSSM-7054

Link to docs preview:
https://80723--ocpdocs-pr.netlify.app/openshift-enterprise/latest/service_mesh/v2x/servicemesh-release-notes

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by pbajjuri20 — APPROVED** [review]
Verified the Component versions and know Issues. 

LGTM..!

**Comment by gwynnemonahan** [comment]
/label peer-review-needed 

Now that all tests have passed!

File: modules/ossm-release-2-4-10.adoc [hunk 1]
```diff
@@ -0,0 +1,29 @@
+////
+Module included in the following assemblies:
+* service_mesh/v2x/servicemesh-release-notes.adoc
+////
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-release-2-4-10_{context}"]
+= {SMProductName} version 2.4.10
+
+This release of {SMProductName} addresses Common Vulnerabilities and Exposures (CVEs), and is supported on {product-title} 4.12 and later.
```
Comment (Srivaralakshmi) [inline modules/ossm-release-2-4-10.adoc:10]:
```suggestion
This release of {SMProductName} addresses Common Vulnerabilities and Exposures (CVEs) and is supported on {product-title} 4.12 and later.
```

File: modules/ossm-release-2-5-4.adoc [hunk 2]
```diff
@@ -0,0 +1,26 @@
+////
+Module included in the following assemblies:
+* service_mesh/v2x/servicemesh-release-notes.adoc
+////
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-release-2-5-4_{context}"]
+= {SMProductName} version 2.5.4
+
+This release of {SMProductName} addresses Common Vulnerabilities and Exposures (CVEs), and is supported on {product-title} 4.12 and later.
```
Comment (Srivaralakshmi) [inline modules/ossm-release-2-5-4.adoc:10]:
```suggestion
This release of {SMProductName} addresses Common Vulnerabilities and Exposures (CVEs) and is supported on {product-title} 4.12 and later.
```

File: modules/ossm-release-2-6-1.adoc [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+|1.28.5
+
+|Kiali
+|1.89.1
+|===
+
+[id="ossm-fixed-issues-2-6-1_{context}"]
+== Fixed issues
+
+* https://issues.redhat.com/browse/OSSM-6766[OSSM-6766] Previously, the {SMPlugin} failed if the user wanted to update a namespace (for example, enable/disable injection), or create any Istio object (for example, create traffic policies). Now, the user can successfully update a namespace and create any Istio object.
```
Comment (Srivaralakshmi) [inline modules/ossm-release-2-6-1.adoc:35]: I suggest rewriting this for better readability.

**Review by Srivaralakshmi — COMMENTED** [review]
@gwynnemonahan Nice work! Left some comments. PTAL. Thanks!

File: modules/ossm-release-2-6-1.adoc [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+|Component |Version
+
+|Istio
+|1.20.8
+
+|Envoy Proxy
+|1.28.5
+
+|Kiali
+|1.89.1
```
Comment (mkralik3) [inline modules/ossm-release-2-6-1.adoc:29]: Here should be also 1.73.14, the 2.6 will still use Kiali server 1.73 even though we release the newer operator

**Review by mkralik3 — CHANGES_REQUESTED** [review]

File: modules/ossm-release-2-6-1.adoc (hunk 4 above)
Comment (gwynnemonahan) [reply modules/ossm-release-2-6-1.adoc:29]: SMCP V2.6 :     Kiali  1.89.1 Kiali ossmc 1.89.0 was in the Jira so I had update this accordingly based on the Jira.

I see that has changed now so I will make this update.

**Comment by gwynnemonahan** [comment]
Rebased.

**Comment by mkralik3** [comment]
lgtm

**Comment by gwynnemonahan** [comment]
Rebased.

File: modules/ossm-release-2-6-1.adoc [hunk 5]
```diff
@@ -0,0 +1,37 @@
+////
+Module included in the following assemblies:
+* service_mesh/v2x/servicemesh-release-notes.adoc
+////
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-release-2-6-1_{context}"]
+= {SMProductName} version 2.6.1
+
+This release of {SMProductName} includes 2.6.1, 2.5.4, and 2.4.10, addresses Common Vulnerabilities and Exposures (CVEs), contains a bug fix, and is supported on {product-title} 4.14 and later.
```
Comment (jeana-redhat) [inline modules/ossm-release-2-6-1.adoc:10]: With one list item being a list of items, [ISG](https://www.ibm.com/docs/en/ibm-style?topic=punctuation-semicolons#semicolons-between-items-in-a-series) recommends a semicolon for the main list
```suggestion
This release of {SMProductName} includes 2.6.1, 2.5.4, and 2.4.10; addresses Common Vulnerabilities and Exposures (CVEs); contains a bug fix; and is supported on {product-title} 4.14 and later.
```

Alternatively, you could split this into a few sentences that are each less complex

**Review by jeana-redhat — COMMENTED** [review]
Two style nits here - will DM you to see if you wanna get them before or after merge. One is probably applicable to more than this release, so the fix would likely be out of scope.

/remove-label merge-review-in-progress
/remove-label merge-review-needed

**Pull request merged**